pkg/bgp/speaker/log.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2021 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package speaker
+
+import (
+	"github.com/cilium/cilium/pkg/logging"
+	"github.com/cilium/cilium/pkg/logging/logfields"
+)
+
+var log = logging.DefaultLogger.WithField(logfields.LogSubsys, "bgp-spkr")
```

Comment:
nit: 

```suggestion
var log = logging.DefaultLogger.WithField(logfields.LogSubsys, "bgp-speaker")
```

(In general we tend to follow the path under `pkg/` and replace `/` with `-` to determine the subsystem)

Reply:
Thanks, was not aware we had an existing naming scheme for this